webhook.go

```diff
@@ -28,6 +30,43 @@ type DeliveryConfig struct {
 	AlternativeURLs []string `json:"alt_urls,omitempty"`
 }
 
+// WebhookConfig is a Webhook substructure with data related to event delivery.
+type WebhookConfig struct {
+	// Accept is content type of outgoing events. The following content types are supported, otherwise
+	// a 406 response code is returned: application/octet-stream, application/jsonl, application/msgpack.
+	Accept string `json:"accept"`
```

Comment:
note:
multipart response will be used for octet-stream and json
single response will be used for msgpack and jsonl